src/main/java/com/iexec/worker/tee/post/compute/worflow/FlowManager.java

```diff
@@ -88,23 +87,26 @@ public static void signComputedFile(ComputedFile computedFile) {
     }
 
     /*
-     * 4 - copyComputedFileToHost
+     * 4 - sendComputedFileToHost
```

Comment:
Maybe a little comment to explain why we chose to save the result by sending it to the worker.

Reply:
Updated, thank you!